rocAL/rocAL/source/pipeline/master_graph.cpp

```diff
@@ -796,13 +796,78 @@ MasterGraph::copy_out_tensor(void *out_ptr, RocalTensorFormat format, float mult
                     else if(output_data_type == RocalTensorDataType::FP16) {
```

Comment:
Move { in Line 796 to next line to maintain same style as of F32

Reply:
Resolved in latest commit